server/services/appointeeAttendeeService.ts

```diff
@@ -0,0 +1,26 @@
+import { Prisoner } from '../@types/prisonerOffenderSearchImport/types'
+import PrisonService from './prisonService'
+import { toDateString } from '../utils/utils'
+import { ServiceUser } from '../@types/express'
+
+export default class AppointeeAttendeeService {
+  constructor(private readonly prisonService: PrisonService) {}
+
+  async findUnavailableAttendees(attendeePrisonerNumbers: string[], user: ServiceUser): Promise<string[]> {
```

Comment:
Similar to [this](https://github.com/ministryofjustice/hmpps-activities-management-api/blob/542eecae3237e09502583affc1f087be4433e39a/src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsactivitiesmanagementapi/service/AppointmentAttendeeService.kt#L74) in the api code, but no considering expired moves.

NB: Manage Appointee Attendees job will remove any attendees who shouldn't be there anyway.